frontend/apps/pablo/sentry.server.config.js

```diff
@@ -0,0 +1,20 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  environment: process.env.NODE_ENV === "production" ? "Release" : "Preview",
```

Comment:
Thanks, yeah I think it could be done. I've just trimmed the URL reported by vercel to match the constraints of Sentry. It should be testable once we have our preview links up and running now.